VERSION

```diff
@@ -1 +1 @@
-v3.1.1
\ No newline at end of file
+v4.0.0-3-g75181da
```

Comment:
why do we need to bump major version, sounds like a compatible change w/o major features either?